### DISCUSSION TITLE
inconsistent segmentations

### DISCUSSION THREAD

**Discussion by kwarthen** [simnibs/simnibs#261] [7.Creating head models]
Hello, 
I'm running into an odd issue where the segmentations are very inconsistent, where I'll run it on one T1 image and get a poor segmentation (particularly between GM and CSF, no T2 images available), and run it again and it will be much better. All of the settings that I've checked in settings.ini are the same between runs, could this have to do with computational resources? I'm not sure what else would impact this besides the affine registration to atlas, but I'd think that would be more consistent. 
thanks!

**Comment by oulap** [comment]
That sounds odd indeed. Are you running it on the exact same input scan every time? The affine registration can impact the segmentation quite a bit, but it shouldn't be fairly consistent. Could you maybe share the log files of the two runs?
